img/a/americancentury.com.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="41.968" height="41.968" fill="none"><g fill="#008061"><path d="m30.324 32.705 1.147.753c.883-1.371 2.583-2.704 3.901-3.144l-.156-1.46c-1.856.582-3.817 2.16-4.892 3.851zM27.979 31.765l1.32.477c1.714-2.86 4.545-4.954 7.67-5.556l-.265-1.41c-3.688.74-6.839 3.062-8.725 6.49zM23.287 32.15l1.32-.325c-2.896-9.312-9.786-16.35-19.172-19.13l-.341 1.38c9.207 2.64 15.65 9.889 18.193 18.075zM15.71 32.233l1.347-.391a16.263 16.263 0 0 0-11.852-9.309l-.193 1.387a14.876 14.876 0 0 1 10.698 8.313z"/><path d="m5.151 25.59-.2 1.4c3.279.49 6.137 2.392 7.923 5.246l1.26-.59C12.14 28.352 8.89 26.153 5.15 25.59zM6.206 28.842l-.169 1.401c1.913.516 3.611 1.7 4.825 3.349l1.157-.689c-1.43-2.027-3.491-3.468-5.813-4.061zM26.422 29.435l-.489-1.276c2.914-4.503 7.583-7.468 12.738-8.149l-.345-1.402c-5.202.736-9.933 3.632-13.056 8.026l-.573-1.18c3.236-4.551 7.996-7.645 13.281-8.667l-.336-1.371c-5.393 1.067-10.27 4.152-13.68 8.69l-.764-1.274c3.417-4.384 8.06-7.519 13.265-9.013l-.45-1.362c-5.322 1.546-10.078 4.722-13.631 9.148l-.728-.998c3.583-4.483 8.356-7.896 13.667-9.806l-.482-1.347a31.94 31.94 0 0 0-4.677 2.136c.816-1.759 1.807-3.471 2.964-4.912l-1.063-.908c-1.634 2.04-2.956 4.566-3.933 7.052-.606.396-1.195.815-1.772 1.251.952-3.509 2.542-6.94 4.676-9.696l-1.156-.798c-2.575 3.376-4.387 7.657-5.282 11.928-.533.465-1.052.948-1.554 1.449.594-4.818 2.553-9.81 4.647-13.285l-1.19-.728c-2.229 3.725-4.553 9.445-5.006 15.667-.25.284-.494.574-.734.869-.139-.17-.281-.338-.424-.506.21-6.227 2.046-12.475 5.318-17.563L24.44.703c-3.13 4.923-4.981 10.853-5.404 16.85-.108-.11-1.142-1.138-1.142-1.138.468-5.679 2.06-11.141 4.629-15.712L21.32 0c-2.51 4.486-4.122 9.779-4.701 15.306a33.705 33.705 0 0 0-1.064-.85c.55-4.722 1.63-8.998 3.256-12.737L17.646.925a39.424 39.424 0 0 0-1.693 4.615 21.85 21.85 0 0 0-1.946-3.258l-1.031.974a20.46 20.46 0 0 1 2.397 4.415 48.556 48.556 0 0 0-.57 2.612c-.739-2.274-1.886-4.47-3.398-6.278l-1.023.964c1.883 2.31 3.34 5.53 3.94 8.276l-.036.283c-.49-.338-.988-.665-1.493-.975-.797-2.45-2.316-5.06-3.804-6.833l-.937 1.07c1.092 1.325 2.034 2.92 2.813 4.673a31.477 31.477 0 0 0-4.52-1.93l-.339 1.382c8.756 2.99 16.496 10.145 19.686 20.723l1.138-.049c2.074-4.223 5.888-7.152 10.32-8.062l-.263-1.41c-4.642.959-8.267 3.809-10.465 7.318z"/><path d="M22.68 37.527c-.067-.751-.11-3.203-.128-3.986-2.337-8.767-8.794-15.749-18.415-17.758l-.256 1.388c6.394 1.261 13.95 5.887 16.998 15.545l-.806.29c-2.43-7.514-8.918-13.005-16.53-14.01l-.246 1.4c8.94 1.141 15.034 8.77 15.658 14.338.045.4.066 2.271-.009 2.973-.155 1.452-.654 1.669-.964 1.854-1.283.771-4.861 1.168-4.861 1.168v1.24c1.796-.008 4.6-.3 8.005-.3 3.394 0 6.127.296 7.7.296v-1.236s-2.851-.518-4.861-1.166c-1.112-.478-1.18-.864-1.285-2.036z"/></g></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 41.97 41.97"><g fill="#008061"><path d="m30.32 32.7 1.15.76a8.56 8.56 0 0 1 3.9-3.15l-.15-1.46a9.8 9.8 0 0 0-4.9 3.85zm-2.34-.94 1.32.48a11.75 11.75 0 0 1 7.67-5.55l-.27-1.41a12.8 12.8 0 0 0-8.72 6.49zm-4.7.39 1.33-.33c-2.9-9.3-9.79-16.35-19.18-19.13l-.34 1.39c9.21 2.63 15.65 9.88 18.2 18.07zm-7.57.08 1.35-.39a16.26 16.26 0 0 0-11.85-9.3L5 23.91a14.88 14.88 0 0 1 10.7 8.31z"/><path d="m5.15 25.59-.2 1.4a11.3 11.3 0 0 1 7.92 5.25l1.26-.6c-1.99-3.29-5.24-5.49-8.98-6.05zm1.06 3.25-.17 1.4a8.9 8.9 0 0 1 4.82 3.35l1.16-.69a10.25 10.25 0 0 0-5.81-4.06zm20.21.6-.49-1.28A18.05 18.05 0 0 1 38.67 20l-.34-1.4a19.4 19.4 0 0 0-13.06 8.02l-.57-1.18c3.23-4.55 8-7.64 13.28-8.66l-.34-1.37a22.67 22.67 0 0 0-13.68 8.69l-.76-1.28a25.89 25.89 0 0 1 13.26-9.01l-.45-1.36a27.23 27.23 0 0 0-13.63 9.14l-.73-1a31.02 31.02 0 0 1 13.67-9.8l-.48-1.35a31.94 31.94 0 0 0-4.68 2.14c.82-1.76 1.8-3.47 2.97-4.91l-1.07-.91a27.62 27.62 0 0 0-3.93 7.05c-.6.4-1.2.82-1.77 1.25.95-3.5 2.54-6.94 4.67-9.7l-1.15-.8a30.23 30.23 0 0 0-5.28 11.94c-.54.46-1.06.94-1.56 1.45A35.8 35.8 0 0 1 27.7 3.67l-1.2-.73a36.99 36.99 0 0 0-5 15.67c-.25.28-.5.57-.73.87a33.4 33.4 0 0 0-.42-.5A35.03 35.03 0 0 1 25.65 1.4l-1.21-.7a36.46 36.46 0 0 0-5.4 16.84l-1.15-1.13A38.83 38.83 0 0 1 22.52.7l-1.2-.7a40.1 40.1 0 0 0-4.7 15.3 33.7 33.7 0 0 0-1.07-.84c.55-4.73 1.63-9 3.26-12.74l-1.16-.8a39.42 39.42 0 0 0-1.7 4.62 21.85 21.85 0 0 0-1.94-3.26l-1.03.98a20.46 20.46 0 0 1 2.4 4.41 48.56 48.56 0 0 0-.58 2.61 19 19 0 0 0-3.4-6.27l-1.02.96a21.06 21.06 0 0 1 3.94 8.28l-.03.28a33.5 33.5 0 0 0-1.5-.98A22.01 22.01 0 0 0 9 5.72l-.94 1.07a20.36 20.36 0 0 1 2.82 4.67 31.48 31.48 0 0 0-4.52-1.93L6 10.92a31.1 31.1 0 0 1 19.68 20.72l1.14-.05a14.83 14.83 0 0 1 10.32-8.06l-.26-1.41c-4.65.96-8.27 3.8-10.47 7.31z"/><path d="M22.68 37.53c-.07-.75-.11-3.2-.13-3.99-2.34-8.77-8.8-15.75-18.41-17.76l-.26 1.4a22.3 22.3 0 0 1 17 15.54l-.8.29A20.38 20.38 0 0 0 3.53 19l-.24 1.4c8.94 1.14 15.03 8.77 15.65 14.33.05.4.07 2.28 0 2.98-.16 1.45-.66 1.67-.97 1.85-1.28.77-4.86 1.17-4.86 1.17v1.24c1.8 0 4.6-.3 8-.3 3.4 0 6.13.3 7.7.3v-1.24s-2.85-.52-4.86-1.17c-1.1-.47-1.18-.86-1.28-2.03z"/></g></svg>
```

Here's an SVG that uses `viewBox`

Reply:
Thanks!

How'd you get that down in size?

I'd used Inkscape to edit and save the initial SVG from the American Century website, then I'd used [SVGOMG](https://jakearchibald.github.io/svgomg/) to shrink it, per [CONTRIBUTING.md](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md).

Reply:
I used [SVGOMG](https://jakearchibald.github.io/svgomg/) and turned on every switch, except for "Remove xmlns". Then I adjusted the precision values to make them as low as possible while retaining the same graphic. For this SVG, I ended up with a number precision of 2 and a transform precision of 0.